### PR TITLE
Added functionality to switch to an existing R terminal

### PR DESCRIPTION
**Pull request by gowerc**
**What problem did you solve?**

#337  - Though feel free to reject if you don't like the proposal :) 

**(If you have)Screenshot**

### CODE REVIEW

File: src/rTerminal.ts [hunk 1]
```diff
@@ -71,6 +71,13 @@ export async function chooseTerminal(active: boolean = false) {
             if (rTermNameOpinions.includes(activeTerminalName)) {
```
Comment (renkun-ken) [inline src/rTerminal.ts:71]: Is it `rTermNameOpinions` or `rTermNameOptions`? Looks like a typo introduced a while ago? Or I'm missing something?

Comment (andycraig) [reply src/rTerminal.ts:71]: Definitely a typo. (Possibly by me.)

**Comment by gowerc** [comment]
@andycraig  & @renkun-ken 

Added a commit to fix the typo you mentioned.
Also added a commit to try to select the most "recent" R terminal. (more details in #337 )

**Comment by renkun-ken** [comment]
What happens if there are multiple open terminals but none is named as `['R', 'R Interactive']`? It would be nice to show a message or create a new terminal in this case?

**Comment by gowerc** [comment]
@renkun-ken  -  If there aren't any terminals called R then rTerm is left undefined `undefined` and gets picked up by the block [here](https://github.com/Ikuyadeu/vscode-R/blob/ffeab34ec5bf7fe7131fb75aae6d18ce44ebf087/src/rTerminal.ts#L98) and thus attempts to create a new terminal (printing an error message if its unable to do so).

**Comment by renkun-ken** [comment]
Thanks, just forgot the case has been handled previously.

LGTM.

**Pull request merged**